chart/templates/replicated-supportbundle.yaml

```diff
@@ -19,9 +19,36 @@ stringData:
     spec:
       collectors:
         - logs:
-            collectorName: replicated-logs
+            collectorName: replicated-sdk-logs
```

Comment:
We've transitioned back to `replicated` for the name of the deployed service, so let's keep that for consistency, if possible.